### ISSUE TITLE
AG WG Review: 2.4.8 Location (Level AAA) WCAG2ICT Guidance

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#915]
## Summary
Review of proposed WCAG2ICT guidance for the Level AAA WCAG SC: 2.4.8 Location, as incorporated into the Editor's draft. 

## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on this issue or by creating a PR on the WCAG2ICT document.  
* Level A and AA SCs can be found in the [comments-by-guideline-and-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-by-guideline-and-success-criterion.md) file.
* Level AAA SCs can be found in the [comments-on-level-AAA-success-criteria.md](https://github.com/w3c/wcag2ict/blob/main/comments-on-level-AAA-success-criteria.md)
----

## Background information 

Once finalized, this new WCAG2ICT Note will build on the previous version by also covering WCAG 2.2 level AAA success criteria (currently only level A and AA criteria are covered.  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

* Section [Applying SC 2.4.8 Location to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-sc-2-4-8-location-to-non-web-documents-and-non-web-software).  

## Reference Material
In case you need them, links to various resources:

* [2.4.8 Location](https://www.w3.org/TR/WCAG22/#location) SC text in WCAG 2.2
* [Intent section of Understanding 2.4.8 Location](https://www.w3.org/WAI/WCAG22/Understanding/location#intent)

**Comment by maryjom** [comment]
Two editorial comments: 
- Missing "a" in the word replacement description so it reads "set of non-web documents or a set of non-web software programs".
- Note 3 should use "success criterion" instead of "SC" to be consistent with the rest of the document. You may want to scan the whole document for use of "SC".

**Comment by pday1** [comment]
Current guidance from editor's draft:

> [Applying SC 2.4.8 Location to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-sc-2-4-8-location-to-non-web-documents-and-non-web-software)
> 
> This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.4.8](https://www.w3.org/WAI/WCAG22/Understanding/location.html#intent), replacing “set of Web pages” with “set of non-web documents or set of non-web software programs".
> 
> With these substitutions, it would read:
> 
> 2.4.8 Location: Information about the user's location within a [[set of non-web documents](https://w3c.github.io/wcag2ict/#set-of-documents) or a [set of non-web software programs](https://w3c.github.io/wcag2ict/#set-of-software-programs)] is available.
> 
> NOTE 1 (ADDED)
> See [set of documents](https://www.w3.org/TR/wcag2ict-22/#set-of-documents) and [set of software programs](https://www.w3.org/TR/wcag2ict-22/#set-of-software-programs) in the Key Terms section to determine when a group of documents or software programs is considered a set for this success criterion. Those implementing this document (WCAG2ICT) will need to consider if this success criterion is appropriate to apply to non-web documents and non-web software. See the [Interpretation of Web Terminology in a Non-web Context](https://www.w3.org/TR/wcag2ict-22/#interpretation-of-web-terminology-in-a-non-web-context).
> 
> NOTE 2 (ADDED)
> The accessibility barrier this SC addresses for websites is extremely unlikely in the context of non-web documents and non-web software.
> NOTE 3 (ADDED) (FOR NON-WEB SOFTWARE)
> Sets of software that meet this definition appear to be extremely rare.
> NOTE 4 (ADDED) (FOR NON-WEB SOFTWARE)
> For general guidance, see also the [comments on closed functionality](https://w3c.github.io/wcag2ict/#comments-on-closed-functionality). For commentary on the individual success criterion, see [Success criteria problematic for closed functionality, Location](https://w3c.github.io/wcag2ict/#problematic-for-closed-location).

Proposed content (changes in **bold**):


Missing "a" in the word replacement description so it reads "set of non-web documents or a set of non-web software programs".
Note 3 should use "success criterion" instead of "SC" to be consistent with the rest of the document. You may want to scan the whole document for use of "SC".


> [Applying SC 2.4.8 Location to non-web documents and non-web software](https://w3c.github.io/wcag2ict/#applying-sc-2-4-8-location-to-non-web-documents-and-non-web-software)
> 
> This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.4.8](https://www.w3.org/WAI/WCAG22/Understanding/location.html#intent), replacing “set of Web pages” with “set of non-web documents or **a** set of non-web software programs".
> 
> With these substitutions, it would read:
> 
> 2.4.8 Location: Information about the user's location within a [[set of non-web documents](https://w3c.github.io/wcag2ict/#set-of-documents) or a [set of non-web software programs](https://w3c.github.io/wcag2ict/#set-of-software-programs)] is available.
> 
> NOTE 1 (ADDED)
> See [set of documents](https://www.w3.org/TR/wcag2ict-22/#set-of-documents) and [set of software programs](https://www.w3.org/TR/wcag2ict-22/#set-of-software-programs) in the Key Terms section to determine when a group of documents or software programs is considered a set for this success criterion. Those implementing this document (WCAG2ICT) will need to consider if this success criterion is appropriate to apply to non-web documents and non-web software. See the [Interpretation of Web Terminology in a Non-web Context](https://www.w3.org/TR/wcag2ict-22/#interpretation-of-web-terminology-in-a-non-web-context).
> 
> NOTE 2 (ADDED)
> The accessibility barrier this SC addresses for websites is extremely unlikely in the context of non-web documents and non-web software.
> NOTE 3 (ADDED) (FOR NON-WEB SOFTWARE)
> Sets of software that meet this definition appear to be extremely rare.
> NOTE 4 (ADDED) (FOR NON-WEB SOFTWARE)
> For general guidance, see also the [comments on closed functionality](https://w3c.github.io/wcag2ict/#comments-on-closed-functionality). For commentary on the individual success criterion, see [Success criteria problematic for closed functionality, Location](https://w3c.github.io/wcag2ict/#problematic-for-closed-location).
